pkg/kubeinteraction/secrets.go

```diff
@@ -99,13 +94,18 @@ func (k Interaction) GetSecret(ctx context.Context, secretopt GetSecretOpt) (str
 	return string(secret.Data[secretopt.Key]), nil
 }
 
-func getBasicAuthSecretName(runEvent *info.Event) string {
-	return fmt.Sprintf(basicAuthSecretName, strings.ToLower(runEvent.Organization), strings.ToLower(runEvent.Repository))
+func GetBasicAuthSecretName(e *info.Event) string {
+	if e.BasicAuthSecet == "" {
```

Comment:
one event can have multiple pipelinerun which means multiple secrets
doesn' this need to be an array?

Reply:
you right, I really don't think clearly 😅 

maybe I do it differently then.. because I don't want another sync.Map :-P

Reply:
while deletion we can fetch the name from pipelinerun ? 🤔

Reply:
yes thats it,using pr generatename as  secretName

Reply:
this is going to be a chicken and egg things, i'll take a step back and properly fix this...

Reply:
> while deletion we can fetch the name from pipelinerun ? 🤔

I really don't want to go in that business :-D

Reply:
just sent a new iteration, i went the kiss route and just store the generate secret from Create and pass it to Delete.. 

am I missing something  ?